node/narwhal/src/primary.rs

```diff
@@ -237,7 +248,7 @@ impl<N: Network> Primary<N> {
         if let Some((_, signatures)) = self.proposed_batch.write().as_mut() {
             // Add the signature to the batch.
             signatures.insert(signature, timestamp);
-            debug!("Added a batch signature from peer '{peer_ip}'");
+            debug!("Added a batch signature {} from peer '{peer_ip}'", batch_id);
```

Comment:
ditto: what's the rationale for `batch_id`?

Reply:
ditto above, this helped establish the order of the log lines.